src/python/sdk/econia_sdk/lib.py

```diff
@@ -3,17 +3,31 @@
 from aptos_sdk.account import Account
 from aptos_sdk.account_address import AccountAddress
 from aptos_sdk.client import RestClient
+from aptos_sdk.async_client import RestClient as AsyncRestClient
 from aptos_sdk.transactions import EntryFunction, TransactionPayload
 
 
 class EconiaClient:
     econia_address: AccountAddress
     aptos_client: RestClient
+    aptos_client_async: AsyncRestClient
     user_account: Account
 
-    def __init__(self, node_url: str, econia: AccountAddress, account: Account):
+    def __init__(
+            self,
+            node_url: str,
```

Comment:
At first glance this all looks non-breaking

Can you confirm?

Reply:
Yes, this is all non-breaking (that was the intent)